lib/src/cli/cli.dart

```diff
@@ -143,8 +143,23 @@ const _ignoredDirectories = {
 };
 
 bool _isPubspec(FileSystemEntity entity) {
-  final segments = p.split(entity.path).toSet();
-  if (segments.intersection(_ignoredDirectories).isNotEmpty) return false;
   if (entity is! File) return false;
   return p.basename(entity.path) == 'pubspec.yaml';
 }
+
+extension on Set<String> {
+  bool excludes(FileSystemEntity entity) {
+    final segments = p.split(entity.path).toSet();
```

Comment:
Take the following case:

`ignore: {'dev/projects/dev/'}`
`entity.path`: `dev/projects/dev/foo`

Segments would equal `{'dev', 'projects', 'foo'}` which means the intersection of `segments` and `ignoredDirectories` would be empty and the intersection of `segments` and `this` would also be empty.

Reply:
Setting `recursive` to `true` and running `very_good packages get --recursive --ignore "dev/projects/dev"` completes as expected.

```dart
for (final value in this) {
  final glob = Glob(value, recursive: true);
  if (glob.matches(entity.path)) {
    return true;
  }
}
```

From the `Glob` docs:
```
If [recursive] is true, this glob matches and lists not only the files and directories it explicitly matches, but anything beneath those as well.
```

Let me know what you think!

Reply:
Never mind actually, I forgot to add `/` to the end of `"dev/projects/dev"`.